processing file asmFiles/branches.asm (1 of 27)
processing file asmFiles/count_days.asm (2 of 27)
processing file asmFiles/dual.llsc.asm (3 of 27)
--- memsim.hex	2015-11-21 18:49:20.970894174 -0500
+++ memcpu.hex	2015-11-21 18:49:20.832890342 -0500
@@ -13,7 +13,8 @@
 :04000C00E0430008C5
 :04000D00AC430008F8
 :04000E00FFFFFFFFF2
-:0400200000000002DA
+:0400200000000001DB
+:040022000000008852
 :0400800034011FC068
 :0400810034020090B5
 :040082008C430000AB
processing file asmFiles/dual.mergesort.asm (4 of 27)
processing file asmFiles/fib1.asm (1 of 2)
processing file asmFiles/fib.asm (2 of 2)
processing file asmFiles/jumpandlink.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 3)
processing file asmFiles/mult.asm (2 of 3)
processing file asmFiles/mult_procedure.asm (3 of 3)
processing file asmFiles/orisw.asm (1 of 1)
processing file asmFiles/search.asm (1 of 4)
processing file asmFiles/subroutine_crc.asm (2 of 4)
processing file asmFiles/test.branchjump1.asm (1 of 7)
processing file asmFiles/test.coherence1.asm (2 of 7)
processing file asmFiles/test.ldst1.asm (1 of 3)
processing file asmFiles/test.ldst2.asm (2 of 3)
processing file asmFiles/test.loadstore.asm (3 of 3)
processing file asmFiles/test.rtype.asm (1 of 1)
